client/lib/products-values/constants.js

```diff
@@ -53,7 +53,7 @@ export const JETPACK_BACKUP_PRODUCT_REALTIME_DISPLAY_NAME = (
 		} ) }
 	</Fragment>
 );
-export const JETPACK_BACKUP_PRODUCT_DISPLAY_NAMES = {
+export const JETPACK_PRODUCT_DISPLAY_NAMES = {
```

Comment:
Why we want to go away from the more specific `JETPACK_BACKUP_PRODUCT_DISPLAY_NAMES` constant name here? I guess this should stay as is since we have a few other constants 'namespaced' with `JETPACK_BACKUP_PRODUCT_` prefix.

Reply:
I agree with @delawski on this one. But I'd suggest something else: if you'd like to have a generic one with all Jetpack products, could it be defined like this instead:

```
export const JETPACK_PRODUCT_DISPLAY_NAMES = {
	... JETPACK_BACKUP_PRODUCT_DISPLAY_NAMES,
};
```

Let me know if that makes sense.